packages/common/src/store/pages/chat/slice.ts

```diff
@@ -164,11 +164,13 @@ const slice = createSlice({
         return { ...item, hasTail: hasTail(item, data[index - 1]) }
       })
       // Recalculate hasTail for latest message of new batch
-      if (state.messages[chatId].ids.length > 0) {
+      if (state.messages[chatId] && state.messages[chatId].ids.length > 0) {
```

Comment:
Think I forgot `chatsAdapter.getSelectors` existed when I wrote this, should we be using `selectById` here instead of `state.messages[chatId]`?

Reply:
No because those selectors are from the state root and the reducers are from the slice root